code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -865,3 +865,54 @@
 	)
 	new /obj/structure/bouncy_castle(gored.loc, gored)
 	gored.gib()
+
+/datum/reagent/drug/syndol
+	name = "Syndol"
+	description = "A potent and addictive hallucinogen used by syndicate agents disorient certain targets. \
+		It is said that the hallucinations it causes are tailored to the user's fears, but tests have been inconclusive, \
+		with subjects in security and assistants reporting wildly different experiences."
+	color = "#c90000"
+	taste_description = "metallic"
+	ph = 7
+	overdose_threshold = 10
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
```

Comment:
Nice but how can it be synthesized? I see you didn't add any chemical reaction to create this recipe.

You also didn't add under what categories this reagent should show up in the chem dispenser UI

Reply:
that flag just means that things like bees and the odysseus syringe thingy are able to make more of it

Reply:
yes, also it can appear in strange seed pool